ydb/Cargo.toml

```diff
@@ -19,7 +19,7 @@ once_cell = "1.8"
 prost = "0.10"
 prost-types = "0.10"
 rand = "0.8"
-reqwest= {version="0.11", features=["blocking", "json"]}
+reqwest= {version="0.11", features=["blocking", "json","rustls"]}
```

Comment:
```default-features = false```?

Is your primary goal is enable rustls or disable native-tls? If second - you can add "default-features = false" only, metadata available by simple http without encryption.

Reply:
Ops, my mistake